docs/data-sources/cdn_billing_option.md

```diff
@@ -22,23 +22,32 @@ data "huaweicloud_cdn_billing_option" "test" {
 
 The following arguments are supported:
 
-* `product_type` - (Required, String) Specifies the product mode. Only **base** (basic acceleration) is supported.
+* `product_type` - (Required, String) Specifies the product mode.  
+  Only **base** (basic acceleration) is supported.
 
-* `status` - (Optional, String) Specifies the billing option status. Valid values are **active** (effective) and
-  **upcoming** (to take effect). Defaults to **active**.
+* `service_area` - (Optional, String) Specifies the service area.  
+  The valid values are as follows:
+  + **mainland_china**: Chinese mainland.
+  + **outside_mainland_china**: outside the Chinese mainland.
+  Defaults to **mainland_china**.
```

Comment:
```suggestion
  + **outside_mainland_china**: outside the Chinese mainland.

  Defaults to **mainland_china**.
```
Using this website to validate your document contents:
https://registry.terraform.io/tools/doc-preview